packages/skia/scripts/skia-configuration.ts

```diff
@@ -166,30 +173,52 @@ export const configurations = {
           ["ios_use_simulator", true],
         ],
       },
-      x64: {
+      "x64-iphonesimulator": {
         cpu: "x64",
+        platform: "ios",
         args: [
           ["ios_min_target", iosMinTarget],
-          ["extra_cflags", '["-target", "arm64-apple-ios-simulator"]'],
-          ["extra_asmflags", '["-target", "arm64-apple-ios-simulator"]'],
-          ["extra_ldflags", '["-target", "arm64-apple-ios-simulator"]'],
+          ["extra_cflags", '["-target", "x64-apple-ios-simulator"]'],
+          ["extra_asmflags", '["-target", "x64-apple-ios-simulator"]'],
+          ["extra_ldflags", '["-target", "x64-apple-ios-simulator"]'],
+        ],
+      },
+      "arm64-macosx": {
+        platformGroup: "macosx",
+        cpu: "arm64",
+        platform: "mac",
+        args: [
+          ["extra_cflags", '["-target", "arm64-apple-macosx"]'],
+          ["extra_asmflags", '["-target", "arm64-apple-macosx"]'],
+          ["extra_ldflags", '["-target", "arm64-apple-macosx"]'],
```

Comment:
these should only be necessary for visionos/ios

Reply:
Oh. Does this mean I could also built skia for visionOS here? I looked a bit and it didn't look like gn/ninja had that as a valid platform.